### DISCUSSION TITLE
GSoC topic: Atlas android/ios app

### DISCUSSION THREAD

**Discussion by romain-fontugne** [InternetHealthReport/gsoc#19] [Ideas]
Make an android and/or iOS app that embeds the Atlas software: https://github.com/RIPE-NCC/ripe-atlas-software-probe

**Comment by dpgiakatos** [comment]
For Android, I think we can install the .deb package using Termux:
https://play.google.com/store/apps/details?id=com.termux
https://github.com/termux/termux-app

I haven't tested this workaround.
